### DISCUSSION TITLE
Application got declined, also the error message is not helping.

### DISCUSSION THREAD

**Discussion by seadog007** [community/community#53497] [GitHub Education]
I am trying to apply the student pack, but got reject multiple times.

Every time I got the emails says 
`You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.`
and
`Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.`

I already added my legal name (both English and Chinese) as my profile name, also the academic affiliation document I uploaded has exact the same name as my profile.

I tried 3 different mail that same school issued, also screenshot of school portal, certification of enrollment (both Chinese and English version).
All of them got reject, what should I do next then? 

Also, all of the document in our school is electronic. Should I just print it out and use my camera to take the photo of that and it suddenly become trustworthy? It is ridiculous.

Just let you know how the system is making people frustrating. I am giving up.

**Comment by samus-aran** [comment]
Hi @seadog007 👋 

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍

**Reply by seadog007** [reply]
1. The original document is English
2. The profile name is exact same as my legal name in both English and Chinese
3. Yes, I choose the correct one

None of them works

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
